specs/trust-store-trust-policy.md

```diff
@@ -442,9 +442,9 @@ The RDN consists of an attribute type name followed by an equal sign and the str
   - The value of each `trustedIdentities` list item, if it begins with `x509.subject:`, MUST be followed by comma-separated one or more RDNs.
     Other types of trusted identities may be supported, by using an alternate prefix, or a different format.
     For example, `x509.subject: C=${country}, ST=${state}, L=${locallity}, O={organization}, OU=${organization-unit}, CN=${common-name}`.
-  - Each identity in `identities` list MUST contain country (C), state or province (ST), and organization (O) RDNs.
+  - Each identity in `identities` list MUST contain country (C), state or province (ST or S), and organization (O) RDNs.
     All other RDNs are optional.
-    The minimal possible value is `x509.subject: C=${country}, ST=${state}, O={organization}`,
+    The minimal possible value is `x509.subject: C=${country}, ST=${state}, O={organization}` or `x509.subject: C=${country}, S=${state}, O={organization}`.
```

Comment:
Shouldn't it be` C=${country}, ST=${state}, O={organization}` Or `C=${country},  L=${localityName}, O={organization}` ?

Reply:
This PR is only for the short name `S` that can be used for province/state as well, currently only `ST` is allowed, but `S` is also correct. Using `L` is a different issue, which is tracked here https://github.com/notaryproject/specifications/issues/293, but there is no decision yet.

Reply:
Can you share RFC which defines `S` and is oid for `S` different from `ST` ?

Reply:
@priteshbandi the oid is the same. In the section "6.3.3" of [ITU-X.520](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-X.520-201210-S!!PDF-E&type=items), `S` is used in the example, although `ST` is used for the LDAP name. Another doc also suggests `S` not `ST`, see https://learn.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-certnametostra?redirectedfrom=MSDN#remarks. So, there are some cases that `S` is used. Allowing the use of either 'S' or 'ST' as the field label for State/Province will be beneficial.